code/datums/actions/order_action.dm

```diff
@@ -86,7 +86,14 @@
 		return
 	if(target == src)
 		return
-	var/datum/atom_hud/squad/squad_hud = GLOB.huds[DATA_HUD_SQUAD_TERRAGOV]
+	var/hud_type
+	if(faction == FACTION_TERRAGOV)
+		hud_type = DATA_HUD_SQUAD_TERRAGOV
+	else if(faction == FACTION_TERRAGOV_REBEL)
+		hud_type = DATA_HUD_SQUAD_REBEL
+	else if(faction == FACTION_SOM)
+		hud_type = DATA_HUD_SQUAD_SOM
+	var/datum/atom_hud/squad/squad_hud = GLOB.huds[hud_type]
```

Comment:
hud type can be null here if its not marines/rebel/som, maybe add a crash or default?